glowing_waffles/visualization/aij_plots.py

```diff
@@ -0,0 +1,110 @@
+import matplotlib.pyplot as plt
+
+__all__ = ['seeing_plot']
```

Comment:
```suggestion
__all__ = ['seeing_plot', 'plot_predict_ingress_egress']
```

This ensures that we can set up the import statements elsewhere in the package.